airflow/contrib/hooks/dingding_webhook_hook.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import json
+
+from airflow import AirflowException
+from airflow.hooks.http_hook import HttpHook
+
+
+class DingdingWebhookHook(HttpHook):
+    """
+    This hook allows you to post messages to Dingding use webhook.
+    Dingding token get from conn_id extra field ``webhook_token`` for security.
+
+    For more message detail in
+    `Dingding custom bot <https://open-doc.dingtalk.com/microapp/serverapi2/qf2nxq>`_
+
+    :param http_conn_id: connection that has Dingding ``webhook_token`` in the extra field
+    :type http_conn_id: str
+    :param message_type: Message type you want to send to Dingding
+                         support text/link/markdown/actionCard/feedCard five types so far
+    :type message_type: str
+    :param message: The message you want to send on Dingding
+    :type message: str or dict
+    :param at_mobiles: Users you want to remind at this message.
+    :type at_mobiles: list of str
+    :param at_all: Should remind all people in group or not. If True, will overwrite ``at_mobiles``
+    :type at_all: bool
+    """
+
+    def __init__(self,
+                 http_conn_id='dingding_webhook_default',
+                 message_type='text',
+                 message=None,
+                 at_mobiles=False,
+                 at_all=False,
+                 *args,
+                 **kwargs
+                 ):
+        super(DingdingWebhookHook, self).__init__(*args, **kwargs)
```

Comment:
```suggestion
        super(DingdingWebhookHook, self).__init__(http_conn_id=http_conn_id,*args, **kwargs)
```

Then we don't need the  next line. (See comment later about changing the name of the connection parameter though)

Reply:
Fix